games-action/supertuxkart/supertuxkart-1.0.ebuild

```diff
@@ -0,0 +1,92 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils desktop xdg-utils
+
+DESCRIPTION="A kart racing game starring Tux, the linux penguin (TuxKart fork)"
+HOMEPAGE="https://supertuxkart.net/"
+SRC_URI="mirror://sourceforge/${PN}/SuperTuxKart/${PV}/${P}-src.tar.xz
+	mirror://gentoo/${PN}.png"
+
+LICENSE="GPL-2 GPL-3 CC-BY-SA-3.0 CC-BY-2.0 public-domain ZLIB"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug fribidi +nettle openssl recorder wiimote"
+REQUIRED_USE="^^ ( nettle openssl )"
```

Comment:
`REQUIRED_USE` is irritating for users and I reckon most users would not want Nettle. We should also allow for LibreSSL though. I think you should have:

```bash
IUSE="libressl nettle"

RDEPEND="
	nettle? ( dev-libs/nettle:= )
	!nettle? (
		libressl? ( dev-libs/libressl:= )
		!libressl? ( >=dev-libs/openssl-1.0.1d:0= )
	)
"

...

-DUSE_CRYPTO_OPENSSL=$(usex nettle no yes)
```

Note the subslot operator on Nettle and the min version on OpenSSL.